packages/components/link/stories/link.stories.ts

```diff
@@ -47,7 +47,16 @@ export default {
     },
     highContrast: {
       description:
-        'Sets the link in high contrast mode by disabling the purple color for the visited stated',
+        'Sets the link in high contrast mode by changing the gray underline color',
+      table: {
+        defaultValue: {
+          summary: false,
+        },
+      },
+    },
+    articles: {
```

Comment:
```suggestion
    articles: {
          control: 'boolean',
```

Reply:
Hello @ClaraLpresta,  there is no switch input in storybook when I want to test the prop